blazor/diagram-component/commands.md

```diff
@@ -728,6 +729,249 @@ The following code illustrates how to execute the grouping commands.
 }
 ```
 
+## Z-Order commands
+
+Z – Order commands are used to visually arrange the selected objects such as Nodes, Connectors and Groups on the diagram page with its Z-order values.
```

Comment:
Z – Order commands are used to control the stacking order (Z-Order) of the diagram elements such as nodes, connectors, and groups. This visually arranges the selected diagram objects on the diagram page with their Z-order values.